numba/jitclass/base.py

```diff
@@ -146,6 +146,14 @@ def _fix_up_private_attr(clsname, spec):
         out[k] = v
     return out
 
+def _add_linking_libs(context, call):
+    """
+    Add the required libs for the callable to allow inlining.
+    """
+    cg = context.codegen()
```

Comment:
Instead of adding the libraries to codegen, it should add to the currently active code library `context.active_code_library`.  See https://github.com/numba/numba/pull/3228 for details.